python/pyspark/sql/session.py

```diff
@@ -304,8 +304,11 @@ def __init__(
                 and not self._jvm.SparkSession.getDefaultSession().get().sparkContext().isStopped()
             ):
                 jsparkSession = self._jvm.SparkSession.getDefaultSession().get()
+                self._jvm.SparkSession.applyModifiableSettings(jsparkSession, options)
             else:
                 jsparkSession = self._jvm.SparkSession(self._jsc.sc(), options)
```

Comment:
Shall we add a short comment here that this is the case when we can set static configurations